src/blockrelay/graphene.cpp

```diff
@@ -559,7 +559,7 @@ bool CGrapheneBlock::process(CNode *pfrom,
                         std::find(blockCheapHashes.begin(), blockCheapHashes.end(), coinbase->GetHash().GetCheapHash());
 
                     if (it == blockCheapHashes.end())
-                        throw std::runtime_error("No coinbase transaction found in graphene block");
+                        return error("No coinbase transaction found in graphene block, peer=%s", pfrom->GetLogName());
```

Comment:
this has the same issue...need to use:

LOG(GRAPHENE , "your mesage");
return false;